### DISCUSSION TITLE
[MediaElement] Seeking/Fast forwarding

### DISCUSSION THREAD

**Discussion by EdwardMcFlurry** [CommunityToolkit/Maui#1289] [Q&A]
Hello

How to check if the video was fast forwarded in media element as I tried Seek_Completed() but the breakpoint was not hit when I fast forwarded a video playing in iOS.

I need to take certain actions when and if the User decides to fast forward the video.

Please help out 🙏🏻

**Comment by EdwardMcFlurry** [comment]
> Hello
> 
> How to check if the video was fast forwarded in media element as I tried Seek_Completed() but the breakpoint was not hit when I fast forwarded a video playing in iOS.
> 
> I need to take certain actions when and if the User decides to fast forward the video.
> 
> Please help out 🙏🏻

@jfversluis

**Comment by EdwardMcFlurry** [comment]
> > Hello
> > How to check if the video was fast forwarded in media element as I tried Seek_Completed() but the breakpoint was not hit when I fast forwarded a video playing in iOS.
> > I need to take certain actions when and if the User decides to fast forward the video.
> > Please help out 🙏🏻
> 
> @jfversluis

Anyone??? I event tried v2.0 of this Plugin but cannot seem to get it right. I just want to know if its possible to know whether or not the User has either Fast-forwarded or Rewinded the video via an event handler, that's all 🙏🏻

**Reply by bijington** [reply]
I wouldn't expect the SeekCompleted event to fire when fast forwarding or rewinding as they as simply changing the playback speed. I would suggest looking at the platforms layers to see if they have any events or information around the changing of playback speed.

**Reply by EdwardMcFlurry** [reply]
Hi @bijington

Thank you for getting back to me, the problem of prioritising iOS over Android most of the time is that I saw that this Seek_Completed method did not fire at all as I even created another blank project just to see if the problem persisted in the plugin or not however, I decided to test with Android recently and both the Emulator and Physical device can fire Seek_Completed method, whether I'm changing the playback speed, fast-forwarded or fast-rewinded it did fire of which that was exciting to me BUT now I don't know if iOS is wired that way of perhaps the plugin itself missed the logic on this method. So, I'm only asking if anyone has ever came across this issue. @jfversluis 

Seek_Completed was meant to do something right on both Platforms, so why is it NOT doing anything on iOS both on Simulator and iPhone?

**Reply by EdwardMcFlurry** [reply]
This is the only thing that is blocking me from releasing a new update into Production environment 😓

**Reply by EdwardMcFlurry** [reply]
@bijington 
@jfversluis 

???

**Reply by EdwardMcFlurry** [reply]
@bijington
@jfversluis

???

**Reply by bijington** [reply]
I don't have much knowledge in this area but I assumed that `SeekCompleted` would fire when the user drags the bar to change where in the media they want to be.

You are correct that we would expect that things behave as close as possible on all platforms. I can't guarantee they will of course.

If you are seeing inconsistent behaviour between the seek event firing on different platforms then I would ask that you raise a bug and we go from there 👍

**Reply by jfversluis** [reply]
Just to add to that, I understand you're looking for a solution, but please don't tag me/us in every response and certainly not to force our attention this way. I assure you, that never has the effect that you are after. We're always very willing to help and trying our best to be as active as we can on all channels that we have. Pinging us every other day is not going to make us go any faster. Thanks for understanding!

**Reply by EdwardMcFlurry** [reply]
@bijington thank you so much for taking time to reply as I really appreciate your humble response and guidance.

**Reply by EdwardMcFlurry** [reply]
@jfversluis i hear you, please understand that I and perhaps other people believe that you are thee most suitable person who “might” be familiar with MediaElement and understand completely that you are busy however, I only asked if you might have been aware of the situation at hand on iOS and if not then perhaps advise as to where to go next like @bijington did. 

I’ll create an issue from here on!

**Comment by CliffAgius** [comment]
Closed as answered
